test/connect_test.go

```diff
@@ -87,3 +120,29 @@ func (s *CLITestSuite) TestConnect_Autocomplete() {
 		s.runIntegrationTest(test)
 	}
 }
+
+func (s *CLITestSuite) zipManifest() {
```

Comment:
I did this instead of committing the zip file itself so that any changes to the manifest are easier to parse in the diffs.